src/liblibc/lib.rs

```diff
@@ -6019,7 +6024,9 @@ pub mod funcs {
     }
 
 
-    #[cfg(any(target_os = "windows",
+    //TODO SORT!
```

Comment:
Ah sorry I meant to merge the other commit (in the installer) first, indeed these are things that I am curious if I should do or not. There's instructions for the reviewer under here.